### DISCUSSION TITLE
Why is this project separate from bits-ui?

### DISCUSSION THREAD

**Discussion by Evertt** [huntabyte/shadcn-svelte#588] [General]
From what I can tell, this project is mostly just thinly wrapping bits-ui components. So, why wouldn't you just make bits-ui and shadcn-svelte the same, and just offer multiple ways of "installing" them (one via npm and one in your own `$lib` folder)?

I assume I must be missing something that explains why that's not really possible?

**Comment by mmailaender** [comment]
In my opinion, the differentiation makes sense, although I agree that an explanation would greatly help the intro page. Explaining what is what and when to go with what.

- Bits is a Headless UI Component library
- Shadcn-svelte, on the other hand, is a self-owned styled UI Component library - It ships a design system and opinionated styles on top of the Headless components but in a way that you own them (So it's pretty easy to change the components if you need it).

**Reply by huntabyte** [reply]
This is the answer.  If I made them the same then you'd have to copy the entirety of bits UI into your codebase, then keeping your end updated would be quite the nightmare. Right now, you just have to `pnpm i bits@latest` to get the latest updates for the component functionality such as bug fixes, etc. You can imagine the way you described being quite a PITA.

**Comment by Evertt** [comment]
Thanks to you both, that answered it for me. 🙏
